x-pack/plugins/apm/server/lib/transactions/get_latency_charts/index.ts

```diff
@@ -192,6 +193,7 @@ export async function getLatencyPeriods({
   latencyAggregationType: LatencyAggregationType;
   comparisonStart?: number;
   comparisonEnd?: number;
+  kuery?: string;
```

Comment:
This is a good example where `kuery: string | undefined` would have been better. 
The argument `kuery` should always be supplied to the function, but `undefined` values are allowed.

Reply:
Actually, that wouldn't have made this bug not occur. nvm.